evaluate/eval.py

```diff
@@ -38,8 +38,7 @@ def load_data(self, name, f_path, f_name, get_tid, get_attr, get_val, na_values=
             raw_data['_value_'] = raw_data.apply(get_val, axis=1)
             raw_data = raw_data[['_tid_', '_attribute_', '_value_']]
             # Normalize string to lower-case and strip whitespaces.
-            raw_data['_attribute_'] = raw_data['_attribute_'].apply(lambda x: x.lower())
-            raw_data['_value_'] = raw_data['_value_'].apply(lambda x: x.strip())
+            raw_data['_value_'] = raw_data['_value_'].apply(lambda x: x.lower().strip())
```

Comment:
should we revert this change since it is already being changed in two other PRs?

Reply:
I'll revert this line for now: it should merge fine though as it will display it as a non-change in the other PRs.